src/tree/registries/azure/AzureAccountTreeItem.ts

```diff
@@ -3,22 +3,25 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { AzureAccountTreeItemBase } from "@microsoft/vscode-azext-azureutils"; // This can't be made lazy, so users of this class must be lazy
+import { AzExtParentTreeItem, AzExtTreeItem, IActionContext, ISubscriptionContext } from "@microsoft/vscode-azext-utils";
 import { Disposable } from "vscode";
-import { AzExtParentTreeItem, AzExtTreeItem, AzureAccountTreeItemBase, IActionContext, ISubscriptionContext } from "vscode-azureextensionui";
 import { AzureAccountExtensionListener } from "../../../utils/AzureAccountExtensionListener";
+import { getAzSubTreeItem } from "../../../utils/lazyPackages";
 import { ICachedRegistryProvider } from "../ICachedRegistryProvider";
 import { IRegistryProviderTreeItem } from "../IRegistryProviderTreeItem";
 import { getRegistryContextValue, registryProviderSuffix } from "../registryContextValues";
-import { SubscriptionTreeItem } from "./SubscriptionTreeItem";
+import type { SubscriptionTreeItem } from "./SubscriptionTreeItem";
 
 export class AzureAccountTreeItem extends AzureAccountTreeItemBase implements IRegistryProviderTreeItem {
```

Comment:
Because this class extends one from the `@microsoft/vscode-azext-azureutils` package, it cannot lazy-load it (as far as I know). As a result, we have to make the rest of the Docker extension lazy-load `AzureAccountTreeItem`.